frontEnd/src/components/Footer.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-footer
+    color="primary"
+    padding-left="1rem"
+    padding-right="1rem"
+  >
+    <v-row
+      justify="center"
+      no-gutters
+    >
+    <v-col
+      align="left">
+      <v-btn
+        v-for="link in links"
+        :key="link"
+        :href="link.href"
+        text
+        rounded
+      >
+        {{ link.text }}
+      </v-btn>
+    </v-col>
+    <v-col
+        text
+        font-family='Roboto, sans-serif'
+        rounded
+        align="right"
+        class="primary"
+      >
+      © 2020 - Big Brothers Big Sisters Calgary
```

Comment:
Try <span class-"my-auto"> © 2020 - Big Brothers Big Sisters Calgary </span>

Reply:
did not seem to do the trick sadly :(. Did the best I can for now if it is an issue for the client will fix later.

Reply:
style="margin: auto 0 auto 0" should work instead. "my-auto" is just a class with that style (but I can't get the vuetify CSS library working).

Reply:
my-auto stands for margin-y-axis-auto. There is a similar one for centering stuff horizontally (mx-auto)